ua-snap/arctic-eds
# Jumps to report section upon map click, lat / lng pick, or place name choice

Updates controllers and reports to properly show the report and map search tools upon map click, lat / lng pick, or place name choice.

Testing:
- Go to any plate, open a report via map click and ensure you are taken to report section
- Go to any plate, open a report via lat / lng selection and ensure you are taken to report section
- Go to any plate, open a report via place selection and ensure you are taken to report section
- Using the Close button jumps you to the search bars at the top of the map.

Closes #93
